multichain-testing/package.json

```diff
@@ -40,7 +40,6 @@
     "patch-package": "^8.0.0",
     "starshipjs": "2.4.1",
     "ts-blank-space": "^0.4.4",
-    "tsx": "^4.15.6",
```

Comment:
There's 3 more `bin/tsx` calls in the `multichain-testing/Makefile` as well:
https://github.com/Agoric/agoric-sdk/blob/a99e78960229f8d8e2f98f1d6a7b2405138d65bb/multichain-testing/Makefile#L79-L80

Reply:
whups thanks. I'll make those run with the shebang